### ISSUE TITLE
Better pods navigation under the Workloads section

### ISSUE THREAD

**Issue by joseluisq** [aptakube/aptakube#280]
### What problem are you trying to solve? Do you a suggestion on how to solve it?

Once you search for a pod, for example by clicking on `Workloads / Pods / pod-123-abc` (Overview or another tab) then if you want to reach its **Deployment** (unless you reached that pod via the `Deployments` side menu), you can not easily go up in the navigation path and reach its _Deployment_ (E.g. `deployment-xyz`).
Instead (I couldn't find an easy way), you are forced to go to the _Deployments_ side menu and then search for your pod manually. The same situation applies if wanting to reach a Pod's `Namespace`.

**Suggestion:**

What could be very useful and time-saving is to improve the navigation path in some way.
For example by linking the path parts and making them clickable instead of only showing them up as plain text for a path like e.g. `namespace-abc / deployment-xyz / pod-123-abc` or constructing a better way to navigate easily from a pod detail and possibly applying the same logic to other sections as well.

**Comment by marekjelen** [comment]
At the moment, when the user is on the pod's detail, there is `Owner` that links to RepicaSet/SatefulSet/DaemonSet.

Would it be possible to add, e.g. next to it, it's owner in case there s one, so that we could easily travers up to Deployment in case of a RaplicaSet that is managed by one?

/cc @goenning

**Comment by goenning** [comment]
But the ReplicaSet already links to the deployment, right? So you can go Pod -> ReplicaSet -> Deployment

**Comment by marekjelen** [comment]
Yes, I can click twice in the same and get there. I know this is a minor thing, though what I mean is that we could go straight to Deployment from the Pod and not through the ReplicaSet.
